Editor/AGS.Types/Font.cs

```diff
@@ -19,14 +19,16 @@ public class Font
         private int _sizeMultiplier = 1;
         private int _verticalOffset;
         private int _lineSpacing;
+		private int _autoOutlineThickness = 0;

```

Comment:
we seem to have gratuitous whitespace changes here

Reply:
This file partially has leading tabs, each counting for 4 spaces, and partially leading spaces. This is all mixed throughout the file. And this accounts for the gratuitous whitespace because Github seems to count tabs as 8 spaces.

Speaking of which, I only know the `C++` conventions; what are our current `C#` coding conventions concerning leading whitespace? I plan on doing a change that does nothing but changing **all** of the leading whitespace to our standard, just for this specific file `Font.cs`.

Reply:
In the new C# code we were using 4 space indents, just like in the engine. The most tabs are leftovers of the old CJ's code that has mixed style.

Reply:
Okay, pushing that dedicated whitespace change.

Reply:
> I plan on doing a change that does nothing but changing all of the leading whitespace to our standard

that sounds like a good idea in the first moment, but experience has shown that such huge whitespace-changing commits are very intrusive, it prevents from easily backporting or forward-porting or reversing *any* change that happened before and after such a commit. in my projects i therefore only fix whitespace when it happens to be in an area that needs to be changed during an unrelated commit.

Reply:
I think I would probably agree that large whitespace changes are a bad idea, for the reasons above, but also because a lot of the other Editor source files have mixed whitespace characters and mixed line endings so this is isn't an isolated problem. It would probably be better to find some future event (like the final merge of SDL2 changes or some other milestone release) where everyone is aware that any branches they have might be affected.